.vscode/launch.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
I can also take that back but I find that pretty handy for debugging out of vsc.
And why not check that in...